README.md

```diff
@@ -1,28 +1,51 @@
 Tokend
 =========
 
-_A daemon that interfaces with [Vault][] and [Warden][] to provide a secure method to deliver secrets to servers in the cloud._
+*A daemon that interfaces with [Vault][] and [Warden][] to provide a secure method to deliver secrets to servers in the cloud.*
```

Comment:
Probably worth putting something in here, just like warden, announcing this is a Node.js service.